net/mdns-repeater/+PRE_DEINSTALL.pre

```diff
@@ -0,0 +1 @@
+service mdns-repeater stop
```

Comment:
`; true ` to ensure nothing breaks.

Reply:
BTW: as long as the configuration file is there, it will always start on the next boot anyway.

Reply:
I don't like +PRE_DEINSTALL trickery. Plus, doesn't this stop the service on upgrade/reinstall?